Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
   config.ssh.insert_key = false
-
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 681.2.0"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = "= 647.2.0"
```

Comment:
> It is useful to see when breaking changes are introduced into coreos-beta and coreos-alpha so we can remedy things before they get to stable.

Agreed, but this is intended to be a short-lived bandaid. I'm open to suggestions, but it seems like we need to restrict clusters to 647.2.0 for them to work properly. Or, do we move forward with Vagrant using whatever version it pleases, and living with the caveat that local development workflows may be broken due to Docker 1.6.2?